OpenQA/Exceptions.pm

```diff
@@ -0,0 +1,30 @@
+# Copyright © 2016 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package OpenQA::Exceptions;
```

Comment:
This somehow bothers me. I don't want to introduce OpenQA namespace to os-autoinst unless we are going to make os-autoinst part of the openQA (I don't mean to move the repo, just I consider os-autoinst as independent tool (pending some exceptions like mmapi)).